ColorSupplier.java

```diff
@@ -0,0 +1,8 @@
+import java.util.Random;
+
+public class ColorSupplier {
+    public Colors getRandomColor(){
+       int index = new Random().nextInt(Colors.values().length);
```

Comment:
Please, make `new Random()` class field

Reply:
Could You write some example, please ?

Reply:
`private static Random random = new Random();`
add this field to ColorSupplier class